flink-runtime/src/test/java/org/apache/flink/runtime/scheduler/benchmark/topology/BuildExecutionGraphBenchmarkExecutor.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License
+ */
+
+package org.apache.flink.runtime.scheduler.benchmark.topology;
+
+import org.apache.flink.runtime.executiongraph.ExecutionGraph;
+import org.apache.flink.runtime.executiongraph.TestingExecutionGraphBuilder;
+import org.apache.flink.runtime.jobgraph.JobGraph;
+import org.apache.flink.runtime.jobgraph.JobVertex;
+import org.apache.flink.runtime.scheduler.benchmark.JobConfiguration;
+
+import java.util.List;
+
+import static org.apache.flink.runtime.scheduler.benchmark.SchedulerBenchmarkUtils.createDefaultJobVertices;
+import static org.apache.flink.runtime.scheduler.benchmark.SchedulerBenchmarkUtils.createJobGraph;
+
+/**
+ * The benchmark of building the topology of {@link ExecutionGraph} in a STREAMING/BATCH job. The
+ * related method is {@link ExecutionGraph#attachJobGraph},
+ */
+public class BuildExecutionGraphBenchmarkExecutor {
```

Comment:
I would suggest to unify naming convention with the network benchmarks. So far it was like this:
- `*BenchmarkExecutor` resides in the `flink-benchmarks` repository
- `*Benchmark` class resides in the `flink` repository
so it's the other way compared to this. Reasoning is that the benchmark resides in the `flink` repo, while code executing the benchmark via JMH is in the `flink-benchmarks` repo.

I'm not strongly attached to the current naming scheme, we could modify it, but at least I would keep it the same in all benchmarks.

Reply:
Okay, I'll change the name for unification.